src/core/manager.h

```diff
@@ -448,6 +448,8 @@ struct Manager {
         VarlinkServer *varlink_server;
         /* Only systemd-oomd should be using this to subscribe to changes in ManagedOOM settings */
```

Comment:
maybe update the comment making clear the this is a varlink connection where we are server and oomd connected to us and that this is used only if we run as system manager